zio-http/src/main/scala/zhttp/http/Method.scala

```diff
@@ -3,8 +3,7 @@ package zhttp.http
 import io.netty.handler.codec.http.HttpMethod
 
 sealed trait Method { self =>
-  lazy val asHttpMethod: HttpMethod = Method.asHttpMethod(self)
```

Comment:
What was the motivation for renaming this field?
Tapir depends on this method for its zio-server interpreter and it is now incompatible with the latest tapir.

Reply:
My mistake, I was using an outdated version of tapir.
Still curious why the name was changed `toJava`
thanks.